ocs_ci/ocs/deployment_openshift_logging.py

```diff
@@ -0,0 +1,362 @@
+"""
+This module deploys the openshift-logging on the cluster
+EFK stack
+"""
+
+import logging
+
+from ocs_ci.ocs import constants
+from ocs_ci.ocs.resources.pvc import get_all_pvcs
+from ocs_ci.ocs.resources.pod import get_all_pods
+from ocs_ci.ocs import ocp
```

Comment:
Import `ocp` and `constants` on the same line: `from ocs_ci.ocs import constants, ocs`